docker-compose-configuration/metaphactory-docker-compose/rds-local/.env

```diff
@@ -0,0 +1,21 @@
+#### Project/deployment name ####
```

Comment:
why is this in a sub-sub-folder? >This should be directly under `docker-compose-configuration` (and event that should be shortend to just `docker-compose`. Also the links in the README need to be updated accordingly)

Reply:
done.